Makefile

```diff
@@ -127,7 +133,7 @@ requirements: virtualenv $(REQUIREMENTS)
 	@echo
 	for req in $(REQUIREMENTS); do \
 		echo "Installing $$req..." ; \
-		. $(VIRTUALENV_DIR)/bin/activate && pip install -U -q -r $$req ; \
+		. $(VIRTUALENV_DIR)/bin/activate && pip install $(PIP_OPTIONS) $$req ; \
```

Comment:
Yeah. They better know what they are doing if they are overriding it :P. 

Alternatively, we can make variable changes to just get 'quiet' or not from env variable. Seems less flexible.